Client code generation error

raml file used:
https://github.com/zero-os/0-stor/blob/master/specs/raml/sdstor.raml

command used to generate :
` go-raml client -l go --ramlfile ../../specs/raml/sdstor.raml  --lib-root-urls https://raw.githubusercontent.com/Jumpscale-Cockpit/raml-definitions/master/ --import-path github.com/zero-os/stor/clients/go`

Client code generation output :

```
INFO[2017-06-13T13:12:28+02:00] generating file librairies/reservation/oauth2_itsyouonline_middleware.go 
INFO[2017-06-13T13:12:28+02:00] generating file client_utils.go              
ERRO[2017-06-13T13:12:28+02:00] Error running go fmt on 'client_utils.go' failed:
client_utils.go:21:9: expected ')', found 'INT' 0
client_utils.go:23:2: expected declaration, found 'if'
 
ERRO[2017-06-13T13:12:28+02:00] go fmt failed  
```

client_utils.go : 
```go
// do HTTP request with request body
func (c 0-Stor)doReqWithBody(method, urlStr string, data interface{}, headers, queryParams map[string]interface{}) (*http.Response, error) {
	body, err := encodeBody(data)
	if err != nil {
		return nil, err
	}
    return c.doReq(method, urlStr, body, headers, queryParams)
}

// do http request without request body
func (c 0-Stor)doReqNoBody(method, urlStr string, headers, queryParams map[string]interface{}) (*http.Response, error) {
    return c.doReq(method, urlStr, nil, headers, queryParams)
}

func (c 0-Stor)doReq(method, urlStr string, body io.Reader,headers, queryParams map[string]interface{}) (*http.Response, error) {
	// create the request
	req, err := http.NewRequest(method, urlStr, body)
	if err != nil {
		return nil, err
	}
    req.URL.RawQuery = buildQueryString(req, queryParams)

    if c.AuthHeader != "" {
        req.Header.Set("Authorization", c.AuthHeader)
    }
	for k, v := range headers {
			 req.Header.Set(k, fmt.Sprintf("%v",v))
	}

	return c.client.Do(req)
}
```

--- a/codegen/golang/util.go
+++ b/codegen/golang/util.go
@@ -46,12 +46,12 @@ func setRootImportPath(importPath, targetDir string) string {
 
 // escape identifier to meet this spec
 // https://golang.org/ref/spec#Identifiers
-// - firstChar need to be letter
+// - firstChar need to be letter, prepend "The_" if it is not
 // - replace "-" with "_"
 func escapeIdentifier(str string) string {
 	r := []rune(str)
 	if !unicode.IsLetter(r[0]) {
-		r = append([]rune("_"), r...)
+		r = append([]rune("The_"), r...)
 	}
 	str = string(r)
 	return strings.Replace(str, "-", "_", -1)

